Sources/Public/AnyCalendarItemModel.swift

```diff
@@ -41,7 +41,7 @@ public protocol AnyCalendarItemModel {
   /// Compares the view models of two `CalendarItemModel`s for equality.
   ///
   /// - Note: There is no reason to invoke this function from your feature code; it should only be invoked internally.
-  func _isViewModel(equalToViewModelOf other: AnyCalendarItemModel) -> Bool
```

Comment:
renamed the base of this function to `_isViewModelEqual...` rather than just `_isViewModel...`, which I think is the more correct way of naming a swift function.